src/OmniSharp.Roslyn.CSharp/Services/Signatures/SignatureHelpService.cs

```diff
@@ -133,7 +133,15 @@ private IEnumerable<IMethodSymbol> GetMethodOverloads(SemanticModel semanticMode
                 return new IMethodSymbol[] { };
             }
 
-            return symbol.ContainingType.GetMembers(symbol.Name).OfType<IMethodSymbol>();
+            var methodOverloads = symbol.ContainingType.GetMembers(symbol.Name).OfType<IMethodSymbol>();
+            var baseTypeSymbol = symbol.ContainingType.BaseType;
+            while(baseTypeSymbol != null && baseTypeSymbol.ContainingNamespace.Name != "System")
+            {
+                methodOverloads = methodOverloads.Concat(baseTypeSymbol.GetMembers(symbol.Name).OfType<IMethodSymbol>());
```

Comment:
I think this will add inaccessible members with the same name.

Reply:
Yes it is returning all the methods regardless of their accessibility.